1228-public_page/1530-files_with_administrative_agreement_is_not_visible.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Files with Administrative Agreement is not visible
```

Comment:
The Feature and Scenario have the same name, the Feature should probably be called something like "Files that are Administrative Agreements have different visibility based on <something>".

Reply:
Why can't they have the same name when there is only one Scenario in the feature file?

Reply:
One scenario per file? Ok, based on the other response (*any* User), I guess this is fine :)